src/sentry/utils/event_tracker.py

```diff
@@ -0,0 +1,55 @@
+import logging
+from enum import IntEnum
+
+from sentry.utils.hashlib import md5_text
+
+
+class EventStageStatus(IntEnum):
+    REDIS_PUT = 1
+    """
+    redis_put
+
+    save_event_started
+
+    save_event_finished
+
+    snuba_topic_put
+
+    commit_log_topic_put
+
+    ppf_topic_put
+
+    post_process_started
+
+    post_process_finished / the same as redis_deleted
```

Comment:
What does this mean? Mind adding them in the EventStageStatus enum rather than in the comment if that's what it's supposed to represent. How come redis put is 1, does that mean it's the first step in the pipeline that will be recorded?